docs/topics/templates/document-attributes.adoc

```diff
@@ -104,8 +104,8 @@
 :SpringBootMvnPluginVersion: 1.5.14.RELEASE
 
 // used in the BOM file example and in the Vert.x Runtime developer Guide
-:VertXVersion: 3.5.1.redhat-004
-:VertXMvnPluginVersion: 1.0.15
+:VertXVersion: 3.5.3.redhat-00001
```

Comment:
@inoxx03 is `00001` a typo?

Reply:
Do not believe so, that is the version in the BOM though I know it is different from previous releases.

Reply:
@rhoads-zach Can confirm, not a typo. I've checked the version against the BOM file and it checks out. Not sure why the string changed, could be due to tooling updates on the Vertx productization side.

Reply:
thank you @huntley and @inoxx03

Reply:
This is actually a new productization convention. All products will have the numeric suffix in a 5-digit form.